services/web/server/src/simcore_service_webserver/clusters/models.py

```diff
@@ -22,6 +22,32 @@ class Config:
 CLUSTER_NO_RIGHTS = ClusterAccessRights(read=False, write=False, delete=False)
 
 
+class Authentication(BaseModel):
+    type: str
+
+    class Config:
+        extra = Extra.forbid
+
+
+class SimpleAuthentication(Authentication):
+    type: Literal["simple"] = "simple"
+    username: str
+    password: str
+
+
+class KerberosAuthentication(Authentication):
+    type: Literal["kerberos"] = "kerberos"
+
```

Comment:
no data to auth jupyerhub and kerberos?

Reply:
this is temporary for now. I'll add some docs